assets/app/js/controllers/conclusionDialog.js

```diff
@@ -254,7 +269,12 @@ class ConclusionDialog extends Controller {
           multiple: true,
           maximumSelectionLength: 1,
           placeholder: 'Enter language',
-          tags: true
+          tags: true,
+          language: {
+            'noResults': function () {
+              return (`SCWB did not find any primary programming languages in the scan associated with ${conclusion.path}`);
```

Comment:
In some cases, this `conclusion.path` value is very long and is awkward to show in a small menu item. 

Instead of `${conclusion.path}`, we should have "the current path".